src/constants/constants.ts

```diff
@@ -6,13 +6,15 @@ if (process.env.NODE_ENV === 'development') {
 } else {
     APP_URL = `https://${process.env.ENVIRONMENT}.${process.env.DOMAIN_NAME}`;
 }
-const environment: Environment = process.env.ENVIRONMENT ? process.env.ENVIRONMENT as Environment : Environment.LOCALHOST;
+const environment: Environment = process.env.ENVIRONMENT
+    ? (process.env.ENVIRONMENT as Environment)
+    : Environment.LOCALHOST;
 
 const commonConstants = {
     ENVIRONMENT: environment,
     BUILD_DATE: process.env.BUILD_DATE,
     AFTER_LOGIN_URL: `${APP_URL}/afterLogin`,
-    DECIMALS_IN_GEOMETRIES: 8,
+    DECIMALS_IN_GEOMETRIES: 6, // Max value 6 currenlty at joredb columns: numeric(8,6)
```

Comment:
If you have 6 decimals in geometries and then you keep mashing a number that is 5 or larger the value seems to be changing. This probably has nothing to do with the amount of decimals allowed but some bug with the rounding of the number.

Reply:
Will be fixed here: https://github.com/HSLdevcom/jore-map-ui/issues/1322